README.md

```diff
@@ -1,2 +1,39 @@
-# ui-wot
-Independent UI components library to build interfaces for IoT applications
+# UI-WoT
+
+## 👋 Hey there!
+
+Welcome to **UI-WoT**, a Google Summer of Code 2025 project with Eclipse Foundation!. We are working to parse any valid WoT Thing Description, extract all its affordances (properties, actions, and events), and then automatically render them as usable UI widgets, and enable real-time interaction
```

Comment:
We don't need to mention that it is a GSoC project

Reply:
Also it is a UI generator and a library to embed these widgets in different web front ends.